LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/matching/font-unicode-PUA-ref.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
I don't think we want to check in a `-ref.html` file in additional to the `-expected.html`.

Reply:
We do, this is pretty common with WPT reftests